osu.Game/OsuGame.cs

```diff
@@ -483,13 +483,13 @@ public void PerformFromScreen(Action<IScreen> action, IEnumerable<Type> validScr
         /// <param name="retrieveInstance">A function to retrieve a (potentially not-yet-constructed) target instance.</param>
         /// <param name="action">The action to perform on the instance when load is confirmed.</param>
         /// <typeparam name="T">The type of the target instance.</typeparam>
-        private void waitForReady<T>(Func<T> retrieveInstance, Action<T> action)
+        public void WaitForReady<T>(Func<T> retrieveInstance, Action<T> action)
```

Comment:
Not sure about exposing this as public, feels a bit like making `Schedule()` public. Would probably make `protected` and add a specialised method in `OsuGameAndroid` just for that import path (i.e. move the
```csharp
game.WaitForReady(() => game, _ => Task.Run(() => game.Import(stream, cursor.GetString(filename_column))));
```
thing to `OsuGameAndroid` as a public method and give it some reasonable name).